CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+
+### Bug Fix
+
+- If you'd like to submit a fix for a bug, please read the [How To](#how-to-contribute) for how to send a Pull Request.
+- Indicate on the open issue that you are working on fixing the bug and the issue will be assigned to you.
+- Write `Fix #xxxx` in your PR text, where xxxx is the issue number (if there is one).
+- Include a test that isolates the bug and verifies that it was fixed.
+
+### New Features
+
+- If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [GitHub issue].
+- You can also join us on [Discord]() to discuss some initials thoughts.
+- If you'd like to implement the new feature, please wait for feedback from the project maintainers before spending too much time writing the code. In some cases, enhancements may not align well with the project future development direction.
+- If applicable, implement the code for the new feature and please read the [How To](#how-to-contribute).
+
+### Documentation & Miscellaneous
+
+- If you have suggestions for improvements to the documentation or examples (or something else), we would love to hear about it.
+- As always first file a [Github issue].
+- Implement the changes to the documentation, please read the [How To](#how-to-contribute).
+
+## How To Contribute
+
+### Requirements
+
+- Linux, Mac OS X, or Windows.
+- [Git](https://git-scm.com) (used for source version control).
+- An IDE such as [Android Studio](https://developer.android.com/studio) or [Visual Studio Code](https://code.visualstudio.com/).
+
+### Forking & cloning the repository
+
+- Ensure all the dependencies described in the previous section are installed.
+- Fork `https://github.com/invertase/melos` into your own GitHub account. If
+  you already have a fork, and are now installing a development environment on
+  a new machine, make sure you've updated your fork so that you don't use stale
+  configuration options from long ago.
+- `git clone git@github.com:<your_name_here>/melos.git`
+- `git remote add upstream git@github.com:invertase/melos.git` (So that you
+  fetch from the main repository, not your clone, when running `git fetch` or `git pull`
+
+## Environment Setup
+
+### Local development setup
+
+To setup and use this melos mono repo locally for the purposes of contributing, clone it and run the following commands from the root of the repository:
+
+```bash
+# Install melos if it's not already installed:
+dart pub global activate melos
+
+# Activate 'melos' from path:
+melos activate
+
+# Confirm you now using a local development version:
+melos --help
+# You should now see a banner printed at the top of the help output similar to:
+# ---------------------------------------------------------
+# | You are running a local development version of melos. |
+# ---------------------------------------------------------
+```
+
+### Install
+
+```bash
+dart pub global activate melos
+```
```

Comment:
This section is not necessary.

Reply:
This was suggested to be added [here](https://github.com/invertase/melos/issues/394#issuecomment-1270748150)

Reply:
Yeah, that's a bit confusing. The `Install` section should be above the `Local development setup` section in the REAMDE.

Reply:
But in `CONTRIBUTING.md` we don't need the `Install` section.

Reply:
Removed.

Reply:
> Yeah, that's a bit confusing. The `Install` section should be above the `Local development setup` section in the REAMDE.

Should I create a separate issue for it?

Reply:
You can just make that change in this PR. Maybe also replace `Local development setup` with `How to contribute` and link to this new document.

Reply:
Then I shouldn't remove `Local development setup` from `CONTRIBUTING.md`.